fab/fabfile.py

```diff
@@ -483,6 +486,21 @@ def record_successful_deploy(url):
         })
 
 
+@task
+@roles(ROLES_DB_ONLY)
+def set_in_progress_flag():
+    sudo('redis-cli SET {} true'.format(IN_PROGRESS_FLAG))
+
+    # Set deploy flag to expire in 30 minutes
+    sudo('redis-cli EXPIRE {} {}'.format(IN_PROGRESS_FLAG, 60 * 30))
```

Comment:
sure i think we might want to change the flag name then cause it won't necessarily be true